### DISCUSSION TITLE
Getting Started guide

### DISCUSSION THREAD

**Discussion by ghost** [e107inc/e107#4671] [General]
Is there a getting started guide? I like the developer area but that does not cover the normal use. Creating and editing pages and menu's is not as straight forward as expected. For some reason Google is not helping me to find one.

Thanks for your concern

**Comment by Moc** [comment]
Currently there is no such guide, but it is being worked on in the background. Hopefully it will be made public in the coming months. 

In the mean time, is there any specific question you have regarding creating and editing pages/menus?
